src/routes/Products.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { useFetchProducts } from "../store/products/products.hooks";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+
+const Products: React.FC = () => {
```

Comment:
So though we call this a `route` it is also a component?

Reply:
I should probably have called that folder "views" rather than "routes". They're just components like any other, but they're "top-level" components, so to speak.